datasource/index.js

```diff
@@ -184,6 +191,42 @@ module.exports = yeoman.generators.Base.extend({
     }.bind(this));
   },
 
+  installConnector: function() {
```

Comment:
Will this always execute after the selection step? Yeoman has changed slightly since we first used this to guarantee order... wanted to double check.